apps/explorer/src/components/SearchBar.tsx

```diff
@@ -84,11 +86,19 @@ const SearchBar = ({ className }: SearchBarProps) => {
         } else {
           redirect(`/transactions/${word}`)
         }
+      } else if (word.length === 66) {
```

Comment:
I checked at least 10 addresses and they all had public keys of `66` characters. I assume that's the standard length.